draft-chfl-intarea-grim.xml

```diff
@@ -113,7 +139,15 @@ TBA
 <name>
 Terminology
 </name>
-<t>TBA</t>
+<t>Legacy node: a node which does not implement protocol changes or algorithms proposed by this document.</t>
+<t>Updated node: a node which supports large MTU and implements protocol changes and/or algorithms proposed by this document.  </t>
+<t>Large MTU: MTU above 1500 bytes.</t>
+<t>Large MTU link: a link which supports large MTU</t>
+<t>Legacy link: a link which doesn't supports large MTU (a link with MTU equal or lower than  1500 bytes).</t>
+<t>Link MTU: The link Maximum Transmission Unit (MTU) is the size in bytes of the largest IP packet, including the IP header and payload, that can be transmitted over a link. See <xref target="RFC8899"/>.</t>
+<t>MAX_PLPMTU: the largest size of the packet the sender will attempt to use (see  <xref target="RFC8899"/>).</t>
+<t>Least Common MTU: a new term, introduced by this document, referring to a MTU which the sender uses until an explicit signal is received, indicating that a larger MTU can be used.</t>
```

Comment:
I like the quantity, but not the name: "Least Common" to me means "most rare", but then I come from the UK, and we use "Lowest Common" to mean minimum. I could suggest "Default MTU" as another possibility?

Reply:
SGTM, done!